modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/AbstractSnapshotSelfTest.java

```diff
@@ -277,7 +280,8 @@ protected void ensureCacheAbsent(CacheConfiguration<?, ?> ccfg) throws IgniteChe
     protected <K, V> CacheConfiguration<K, V> txCacheConfig(CacheConfiguration<K, V> ccfg) {
         return ccfg.setCacheMode(CacheMode.PARTITIONED)
             .setBackups(2)
-            .setAtomicityMode(CacheAtomicityMode.TRANSACTIONAL);
+            .setAtomicityMode(CacheAtomicityMode.TRANSACTIONAL)
+            .setAffinity(new RendezvousAffinityFunction(false, 4));
```

Comment:
What's wrong with default partitions count? Only 4 partitions looks too small, especially when there are more than 4 nodes in the cluster.